platforms/documentation/docs/src/docs/userguide/glossary.adoc

```diff
@@ -76,6 +135,10 @@ For more information, see the sections on <<feature_variants.adoc#feature_varian
 
 == G
 
+[[sub:gradle_build]]
+Gradle Build::
+A Gradle Build is one or more project(s) that use the Gradle Build Tool to compile, test, package, and deploy code. A Gradle Build should contain at least one Gradle build file and is executed using the `gradle` command, such as `gradle build`.
```

Comment:
❗ I'm going to nitpick harshly here, because this is an important definition that we really should be much more consistent about, as we talk about Gradle builds all the time in docs and error messages.

- "one or more project(s)" - Yes, a build is one or more _Gradle_ projects.  But we all use the shorthand of talking about a "project" as being a single Gradle build.  A single Gradle build is (usually) a single IDE project, too, which is confusing.  As builds/projects can be structured radically differently, I think this definition should be focused not on the structure, but on the goal.
- "to compile, test, package, and deploy code" - not every build does all of these, and none of them are in any way required.
- "A Gradle Build should contain at least one Gradle build file" - _should_ contain a _settings_ file at the root would be better advice.  Though a single _build_ file can still work, it's not recommended.
- "and is executed using the `gradle` command" - it can be, but we recommend executing with the `gradlew` command.  Also, the command to execute it doesn't seem like an important part of the definition.
- There's also the noun/verb confusion - a Gradle build is a thing that you write, and the thing that runs when you invoke "gradle".  In my mind, it would absolutely be worth having 2 different words for these.  Like how we can talk about a `class` vs. an `instance` in OOP when we need to be precise, despise just saying `class` most of the time when the meaning is obvious from the context.

I think this will be a very tricky definition to get right because it's so fundamental.  Reminds me of how it's tough to define a point or a line in Geometry.  I think you should "workshop" this on Slack.  I bet there are others here who have thought of this before and can give you a better starting point.